gatsby/src/pages/404.tsx

```diff
@@ -1,12 +1,39 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
-			<h1>Page not found</h1>
+			<Breadcrumbs>
+				<Breadcrumb to="https://www.nice.org.uk/">NICE</Breadcrumb>
+				<Breadcrumb to="/">CKS</Breadcrumb>
```

Comment:
Can you use a Link for the elementType prop please so it navigates client side?
```suggestion
				<Breadcrumb to="/" elementType={Link}>CKS</Breadcrumb>
```